modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -179,13 +179,7 @@ definition {
 		}
 
 		task ("And given the object definition is imported") {
-			ObjectExportImport.importObject(
-				objectFileName = "ImportedObjectWith100Fields",
-				objectName = "CustomObject187390");
-
-			ObjectPortlet.selectCustomObject(label = "Custom Object 187390");
-
-			ObjectAdmin.publishObjectViaUI();
+			ObjectExportImport.importExportedObject(objectName = "CustomObject187390");
```

Comment:
This part as well, we can combine it to the task above. Remember to change the task name